core.d.ts

```diff
@@ -257,7 +258,7 @@ declare namespace core {
  	If file access is available, it is recommended to use `fromFile()` instead
 
 	@param buffer - It works best if the buffer contains the entire file, it may work with a smaller portion as well
-	@returns The detected file type and MIME type or `undefined` when there was no match.
+	@returns The detected file type and MIME type via a promise, or `undefined` when there was no match.
```

Comment:
> via a promise

This is moot as the return type already says it's a promise.

Reply:
But in the readme I kept the 

> _Returns a `Promise` resolving the _...

as the promise is not obvious